Cargo.toml

```diff
@@ -4,9 +4,13 @@ version = "0.1.19"
 authors = ["Lloyd <lloydmeta@gmail.com>"]
 description = "Frunk provides developers with a number of functional programming tools like HList, Generic, LabelledGeneric, Validated, Monoid, Semigroup and friends."
 license = "MIT"
-homepage = "https://github.com/lloydmeta/frunk"
+documentation = "https://docs.rs/frunk/0.1.19/frunk/"
```

Comment:
It looks like we can leave off the part of the path that deals with versioning (e.g. `https://docs.rs/frunk`) so that we can always just look at the latest published version. A bit less to maintain :)